pkgs/tools/backup/rustic-rs/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, fetchFromGitHub, rustPlatform, stdenv, darwin, installShellFiles }:
```

Comment:
As mentioned, elsewhere in Nixpkgs it's usually done like this

```Suggestion
{ lib, fetchFromGitHub, rustPlatform, stdenv, Security, installShellFiles }:
```

an then explicitly inherited from `darwin` at `callPackage`.

I have no idea why that is (I'd much rather the frameworks were available as regular attrs for `callPackage` magic) but note that this goes against the currently practiced convention.

Reply:
When you said to "inherit darwin from callpackage" I though you mean it exactly like this.

I will take a look into how it is done elsewhere and then try to remember to do it that way on the next bump.

Reply:
I am aware fo the pattern but I don't get why its done. Do people really override frameworks individually? Is this even possible? To me it still looks like cargo culting.

Reply:
As I said, I have no idea why it's done. Just noting that it's the current convention.